src/containers/Notifications.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { prefetch } from 'react-fetcher'
+import { fetchActivity, markActivityRead, markAllActivitiesRead, navigate, thank } from '../actions'
+import { values, sortBy, isEmpty, contains } from 'lodash'
+import cx from 'classnames'
+import ScrollListener from '../components/ScrollListener'
+import A from '../components/A'
+import truncate from 'html-truncate'
+import { present, humanDate } from '../util/text'
+const { array, bool, func, number, object } = React.PropTypes
+
+@prefetch(({ dispatch }) => dispatch(fetchActivity(20, 0)))
+@connect(state => ({
+  activities: sortBy(values(state.activities), ['created_at']).reverse(),
```

Comment:
oh never mind, you're working around the case where a new activity shows up during pagination?